src/main/resources/static/scss/sections/_groupings.scss

```diff
@@ -142,3 +142,8 @@
         }
     }
 }
+
+.form-control {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+}
```

Comment:
Can you check if this did not affect the other input boxes such as 'Filter Members' inside /manage-groupings and 'Filter Groupings' in /admin /membership /grouping pages?

Reply:
It did. They don't have radius on the right corners.
![image](https://user-images.githubusercontent.com/77418067/210458826-cc3c704d-4521-4f01-9a38-b90a3a4e63c7.png)
Would it be alright to add an id to the edit description input box and apply the css to that?

Reply:
In selected-grouping.html, there is a rounded-left Bootstrap class around line 55 that does not seem to work. If changing the Bootstrap does not seem to work, add a class instead of an id with a class name similar to 'description-form-button'.